include/cantera/kinetics/LinearBurkeRate.h

```diff
@@ -0,0 +1,140 @@
+
+#ifndef CT_LINEARBURKERATE_H
+#define CT_LINEARBURKERATE_H
+#include "cantera/kinetics/Arrhenius.h"
+#include <boost/variant.hpp>
+#include "cantera/kinetics/Falloff.h"
+#include "cantera/kinetics/ChebyshevRate.h"
+#include "cantera/kinetics/PlogRate.h"
+
+namespace Cantera
+{
+
+//! Data container holding shared data specific to LinearBurkeRate
+/**
+ * The data container `LmrData` holds precalculated data common to
+ * all `LinearBurkeRate` objects.
+ */
+struct LmrData : public ReactionData
+{
+    LmrData();
+
+    void update(double T, double P) override
+    {
+        ReactionData::update(T);
+        pressure = P;
+        logP = std::log(P);
+    }
+
+    bool update(const ThermoPhase& phase, const Kinetics& kin) override;
+
+    using ReactionData::update;
+
+    //! Perturb pressure of data container
+    /**
+     * The method is used for the evaluation of numerical derivatives.
+     * @param  deltaP  relative pressure perturbation
+     */
+    void perturbPressure(double deltaP);
+
+    void restore() override;
+
+    virtual void resize(size_t nSpecies, size_t nReactions, size_t nPhases) override
+    {
+        moleFractions.resize(nSpecies, NAN);
+        ready = true;
+    }
+
+    void invalidateCache() override
+    {
+        ReactionData::invalidateCache();
+        pressure = NAN;
+    }
+
+    double pressure = NAN; //!< Pressure
+    double logP = 0.0; //!< Logarithm of pressure
+    bool ready = false; //!< Boolean indicating whether vectors are accessible
+    vector<double> moleFractions;
+    int mf_number;
+
+protected:
+    double m_pressure_buf = -1.0;
+};
+
+
+//! Pressure-dependent and composition-dependent reaction rate calculated
+//! according to the reduced-pressure linear mixture rule (LMR-R) developed
+//! at Columbia University. @cite singal2025 [CITATION NOT YET ACTIVE]
+/*!
+ * [ADD IN THE MATHEMATICAL FORMULA]
+ */
+class LinearBurkeRate final : public ReactionRate
+{
+public:
+    //! Default constructor.
+    LinearBurkeRate() = default;
+
+    //! Constructor from Arrhenius rate expressions at a set of pressures
+    explicit LinearBurkeRate(const std::multimap<double, ArrheniusRate>& rates);
+
+    LinearBurkeRate(const AnyMap& node, const UnitStack& rate_units={});
+
+    unique_ptr<MultiRateBase> newMultiRate() const override {
+        return make_unique<MultiRate<LinearBurkeRate, LmrData>>();
+    }
+
+    //! Identifier of reaction rate type
+    const string type() const override { return "linear-burke"; }
+
+    //! Perform object setup based on AnyMap node information
+    /*!
+     *  @param node  AnyMap containing rate information
+     *  @param rate_units  Unit definitions specific to rate information
+     */
+    void setParameters(const AnyMap& node, const UnitStack& rate_units) override;
+
+    void getParameters(AnyMap& rateNode, const Units& rate_units) const;
+    void getParameters(AnyMap& rateNode) const override {
+        return getParameters(rateNode, Units(0));
+    }
```

Comment:
I think you can just implement the version that doesn't take a `Units` object directly and skip providing this other overload of `getParameters`.